gemfiles/aws_30.gemfile

```diff
@@ -11,6 +11,7 @@ gem "webrick"
 gem "puma"
 gem "rack-test"
 gem "simplecov", "~> 0.21.2"
+gem "aws-sdk-core", "= 3.191.6"
```

Comment:
Can we mention somewhere (maybe in the commit message) the reason why we use a strict version constraint for `aws-sdk-core` gem?

Reply:
Wrote in the commit message. Please check.

Reply:
Thank you!